WordPress/gutenberg
# Comments Block: fixed issue with custom font sizes and links color

## What?

This PR fixes an issue where custom font sizes were not showing up properly on the frontend and after a reload they were disappearing from the editor.

It also fixes an issue where link color is not applied on site editor, both in Comment Author and Comment Date block.

Co-created with @c4rl0sbr4v0 

## Why?


So that the user can use custom font sizes in the comment author name, comment date, comment reply link and comment edit link blocks.

## How?

The current implementation of the comments blocks was defining a small size attribute that was overriding any custom font sizes defined by the user or the theme. I removed that definition from the blocks and instead moved it to the template of the comments block since it's a style decision.

## Testing Instructions




1. With TT2 active, insert the Comments block
2. Change the font sizes of the affected blocks to some custom value
3. Save and check the frontend, reload the editor and the font size selected should be the one you see.

1. Change the link color of Comments Date and Comments Author in the Site Editor.
2. Check that changes are applied in both editor and frontend.

## Screenshots or screencast 


Before:

<img width="1001" alt="Screenshot 2022-06-09 at 13 27 36" src="https://user-images.githubusercontent.com/3593343/172836413-81c81a05-c83c-472a-b991-26c38d7c72ea.png">

After:

<img width="839" alt="Screenshot 2022-06-09 at 13 27 00" src="https://user-images.githubusercontent.com/3593343/172836403-6f111540-c8b4-45d5-b40a-5c473c14d946.png">
